app/assets/stylesheets/custom.css.scss

```diff
@@ -28,7 +34,7 @@ body {
 }
 
 .thumbnail {
-  background-color: $light-gray !important;
+  background-color: $primary_color !important;
```

Comment:
!important should not be used<br>Name of variable `primary_color` should be written in all lowercase letters with hyphens instead of underscores